src/employee.js

```diff
@@ -2,18 +2,31 @@
 import React, { useState } from "react";
 import { jsx } from "@emotion/core";
 import "./Employee.css";
+import { blueBright } from "ansi-colors";
```

Comment:
Hi Angie! What do you think if use a hexadecimal code color instead import the color from this library?

Reply:
I will amended that, is it ok to use for the backgroundColor: `"rgba(0,` 0, 0, .5)",`?